source/gui/settingsDialogs.py

```diff
@@ -2050,6 +2050,13 @@ def __init__(self, parent):
 		self.UIAInMSWordCheckBox.SetValue(config.conf["UIA"]["useInMSWordWhenAvailable"])
 		self.UIAInMSWordCheckBox.defaultValue = self._getDefaultValue(["UIA", "useInMSWordWhenAvailable"])
 
+		# Translators: This is the label for a checkbox in the
+		#  Advanced settings panel.
+		label = _("Use UI Automation to access the Windows Console (restart NVDA for changes to take effect)")

```

Comment:
Why is a restart necessary?

Reply:
Changing the setting seems to leave NVDA in an inconsistent state until it's restarted. Maybe because of the `badUIAWindowClassNames` insertion?